solara/website/pages/documentation/api/routing/route.py

```diff
@@ -14,18 +14,16 @@
 @solara.component
 def Page():
     route_current, routes = solara.use_route()
-    with solara.VBox() as main:
-        # solara.Warning("Note the address bar in the browser. It should change to the path of the link.")
-        solara.Markdown("*Click on one of the links below to change the route and see the url in your browser change, and match the route.*")
-        with solara.VBox():
-            for route in routes:
-                with solara.Link(route):
-                    current = route_current is route
-                    if current:
-                        solara.Success(f"You are at solara.Route(path={route.path!r})")
-                    else:
-                        solara.Info(f"Go to solara.Route(path={route.path!r})")
-    return main
+    # solara.Warning("Note the address bar in the browser. It should change to the path of the link.")
```

Comment:
I think the comment can also be removed as a part of this PR as well (@maartenbreddels let me know if you disagree)

```suggestion
```

Reply:
ok